statsmodels/tsa/statespace/representation.py

```diff
@@ -105,6 +105,17 @@ class Representation(object):
     initial_variance : float, optional
         Initial variance used when approximate diffuse initialization is
         specified. Default is 1e6.
+    initialization : {'approximate_diffuse','stationary','known'}, optional
+        Initialization method for the initial state.
+    initial_variance : float, optional
```

Comment:
Good catch. Initial variance is spherical in the approximate diffuse case. You could make them elliptical but I can't really imagine an application for it, since the idea is that it should be "infinite" and we're just approximating infinity.